oozie/oozie.sh

```diff
@@ -125,7 +125,30 @@ function install_oozie() {
     --configuration_file "/etc/hadoop/conf/core-site.xml" \
     --name 'hadoop.proxyuser.oozie.groups' --value '*' \
     --clobber
+    
+  bdconfig set_property \
+    --configuration_file "/etc/oozie/conf/oozie-site.xml" \
+    --name 'oozie.service.HadoopAccessorService.supported.filesystems' --value "hdfs,gs" \
+    --clobber
 
+  bdconfig set_property \
+    --configuration_file "/etc/hadoop/conf/core-site.xml" \
+    --name 'fs.AbstractFileSystem.gs.impl' --value 'com.google.cloud.hadoop.fs.gcs.GoogleHadoopFS' \
+    --clobber
+    
+  bdconfig set_property \
+    --configuration_file "/etc/hadoop/conf/core-site.xml" \
+    --name 'google.cloud.auth.service.account.enable' --value 'true' \
+    --description 'Whether to use a service account for GCS authorization.
+    Setting this property to `false` will disable use of service accounts for
+    authentication.' \
+    --clobber
+  
+  if [ -e /usr/lib/hadoop/lib/gcs-connector.jar ]
+  then
```

Comment:
@medb Can gcs-connector.jar be at any other location?

Reply:
Yes, it could be in the `/usr/local/share/google/dataproc/lib` on some image versions, you can add a check for this:
```bash
    local install_dir="/usr/local/share/google/dataproc/lib"
    if [[ ! -d \$install_dir ]]; then
      install_dir="/usr/lib/hadoop/lib"
    fi
```